tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -241,8 +241,21 @@ def notMatches(option):
 
 def oneOf(options=[]):
     """Validate that value does not exist in the provided options array."""
+    """
```

Comment:
this is in fact not required but can improve the validator so instead of writing all values to be matched, we can add a range

Reply:
could you include tests in `test_validators` for this change?